### ISSUE TITLE
WS updates are not arriving

### ISSUE THREAD

**Issue by AndrejLischishin** [ftexchange/ftx#45]
Sometimes ws updates for orders/fills are not arriving , especially in high volatility times.

**Comment by WiktorJ** [comment]
Facing the same issue. We had to resort to HTTP polling because WebSockets seem not to be reliable enough.

**Comment by bkaftanatii** [comment]
Yeah, had the same issue, last time on 26 of February while testing your api

P.S.: actually there are a lot of questions about your ws api, that makes it difficult to use

**Comment by AndrejLischishin** [comment]
wow, i though i am the only one

**Comment by AndrejLischishin** [comment]
but it seems like a real problem

**Comment by AndrejLischishin** [comment]
so we recently added FTX to out stack, we have some other exchanges already connected and it seems like only FTX has this kind of problems with ws updates

maybe somebodyfrom FTX texh team could clear it up @jmoz @nishadsingh1

**Comment by yuriyarabskyy** [comment]
I'm also facing a similar issue.

**Comment by sleepymatto** [comment]
Same

**Comment by yeetmon** [comment]
is this still an issue? trying to decide whether to use REST or WS

**Comment by mircofx** [comment]
Had issues tonight for the first time. Maybe you should double check, meaning both ws and API poll to check order status
